extensions/amp-date-picker/0.1/amp-date-picker.js

```diff
@@ -157,7 +164,23 @@ const DatePickerEvent = {
  * The size in PX of each calendar day. This value allows the date picker to
  * fit within a 320px wide viewport when fully rendered.
  */
-const DEFAULT_DATE_SIZE = 39;
+const DEFAULT_DAY_SIZE = 39;
+
+/**
+ * This is related to a bug in preact.
```

Comment:
Might reword this slightly. "This is related to a bug in preact-compat impacting 'react-dates' rendering"

Reply:
Fair point, I'll make this more precise